lumicks/pylake/population/dwelltime.py

```diff
@@ -542,6 +546,14 @@ def __init__(
                         f"the number of dwell times provided ({dwelltimes.size})."
                     )
 
+            # A minimum time shorter than the discretization time makes no sense (cannot be
+            # observed under this model)
+            rel_tolerance = 1e-6
+            if np.any(discretization_timestep > (1.0 + rel_tolerance) * min_observation_time):
+                raise ValueError(
+                    "The discretization timestep cannot be larger than the minimum observable time."
+                )
```

Comment:
Could add some more info as we've been doing lately with error messages
```suggestion
            if np.any(discretization_timestep > (1.0 + rel_tolerance) * min_observation_time):
                raise ValueError(
                    f"The discretization timestep ({discretization_timestep:0.2e} sec) cannot be larger "
                    f"than the minimum observable time ({min_observation_time:0.2e} sec)."
                )
```
Might want to think about the units there, could be nicer to show in usec but then it doesn't really match what the user input. Not sure

Reply:
I was doing that originally, but one, both or none of these can actually be an array rather than a number. What should I show in that case, the first violation?